### PR TITLE
Update external-email-forwarding.md

### PR DESCRIPTION
**Pull request by SignorelliDenis**
Adding information that disabling forward through this feature will affect the "ForwardingAddress" which is configured by admins. Customers should be aware about this because all the possibilities that we had before the release of this feature never used to block "ForwardingAddress", just forward configured by users.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/external-email-forwarding.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ For instructions on how to configure these settings, see [Configure outbound spa
 
 - Disabling automatic forwarding will also disable Inbox rules that redirect messages to external addresses.
 
+- Disabling automatic forwarding will also disable forwarding performed by administrators through EAC or powershell to external addresses.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/external-email-forwarding.md]:
```suggestion
- Disabling automatic forwarding will also disable forwarding performed by administrators through the Exchange admin center or PowerShell to external addresses.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks.

**Pull request merged**